programs/compressed-pda/src/compressed_account.rs

```diff
@@ -0,0 +1,209 @@
+use anchor_lang::prelude::*;
+use light_hasher::{Hasher, Poseidon};
+use light_utils::hash_to_bn254_field_size_le;
+
+#[derive(Debug, PartialEq, Default, Clone, AnchorSerialize, AnchorDeserialize)]
+pub struct CompressedAccountWithMerkleContext {
+    pub compressed_account: CompressedAccount,
+    pub index_merkle_tree_account: u8,
+    pub index_nullifier_array_account: u8,
```

Comment:
@ananas-block  should we just clean up the api now? i.e. `merkle_tree_account_index` and `nullifier_array_account_index` 
such that the naming is consistent with _index suffix in leaf_index. 

(and more consistent with use of nullifier queue elsewhere would be to do
`nullifier_queue_index`)

Reply:
sure